tds/routers/external.py

```diff
@@ -102,11 +102,25 @@ def create_publication(
         publication_payload = payload.dict()
         publications = (
             session.query(orm.Publication)
-            .filter(orm.Publication.xdd_uri == publication_payload["xdd_uri"])
+            .filter(
+                and_(
+                    str(publication_payload["xdd_uri"]) == orm.Publication.xdd_uri,
+                    str(publication_payload["title"]) == orm.Publication.title,
+                )
+            )
             .all()
         )
+
         if len(publications) != 0:
-            return Response(status_code=status.HTTP_409_CONFLICT)
+            publication = publications[0].__dict__
+            publication.pop("_sa_instance_state")
```

Comment:
This pop is changing the state on the object too, not just the dict. If you want to only change the new dict, you need to create a copy.